### PR TITLE
python: Set indents to 4-space

### PR DESCRIPTION
**Pull request by phlax**
Signed-off-by: Ryan Northey <ryan@synca.io>

Commit Message: python: Set indents to 4-space
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: tools/protodoc/protodoc.py [hunk 1]
```diff
@@ -61,15 +61,15 @@
...
-  {{security_posture}}
+    {{security_posture}}
 
 .. tip::
-  This extension extends and can be used with the following extension {% if categories|length > 1 %}categories{% else %}category{% endif %}:
+    This extension extends and can be used with the following extension {% if categories|length > 1 %}categories{% else %}category{% endif %}:
 
 {% for cat in categories %}
-  - :ref:`{{cat}} <extension_category_{{cat}}>`
+    - :ref:`{{cat}} <extension_category_{{cat}}>`
```
Comment (phlax) [inline tools/protodoc/protodoc.py:72]: this is wrong

File: tools/protodoc/protodoc.py [hunk 2]
```diff
@@ -121,599 +121,607 @@
...
-    raise ProtodocError(f"\n\nUnable to find extension category:  {extension_category}\n\n")
-  return EXTENSION_CATEGORY_TEMPLATE.render(category=extension_category,
-                                            extensions=sorted(extensions))
+    Returns:
+        RST formatted extension category description.
+    """
+    try:
+        extensions = EXTENSION_CATEGORIES[extension_category]
+    except KeyError as e:
+        raise ProtodocError(f"\n\nUnable to find extension category:    {extension_category}\n\n")
```
Comment (phlax) [inline tools/protodoc/protodoc.py:257]: this is wrong

File: tools/protodoc/protodoc.py [hunk 3]
```diff
@@ -121,599 +121,607 @@
...
+    lines = []
+    for index, field in enumerate(msg.field):
+        field_type_context = type_context.extend_field(index, field.name)
+        leading_comment = field_type_context.leading_comment
+        if hide_not_implemented(leading_comment):
+            continue
+        lines.append('"%s": %s' % (field.name, format_field_type_as_json(type_context, field)))
+
+    if lines:
+        return '.. code-block:: json\n\n    {\n' + ',\n'.join(indent_lines(4,
```
Comment (phlax) [inline tools/protodoc/protodoc.py:322]: this is adding extra space

**Comment by phlax** [comment]
@htuch after a bit of messing around i just let yapf reformat to 4-space indents, so hopefully this one is pretty straight forward

**Comment by phlax** [comment]
even if its huge, sorry

File: source/extensions/filters/network/kafka/serialization/generator.py [hunk 4]
```diff
@@ -4,54 +4,54 @@
 
 
 def generate_main_code(serialization_composite_h_file):
-  """
+    """
   Main code generator.
```
Comment (phlax) [inline source/extensions/filters/network/kafka/serialization/generator.py:8]: this doesnt seem to be properly indented - not a big deal and passes linting but should be properly indented i think

**Comment by phlax** [comment]
the only inconsistency im seeing is the indents on docstrings

on the basis that this is the (reproducible) yapf reformatting and because of its size i would suggest we just apply this, and then im happy to go through and fix the indents on the docstrings

**Review by htuch — APPROVED** [review]
Since this is YAPF generated, I'm just going to merge. Agree we can followup after this. Thanks!

**Pull request merged**